installer_scripts/install-yb-voyager

```diff
@@ -75,6 +77,11 @@ main() {
 			centos_main
 			;;
 		"AlmaLinux")
+			sudo yum -qy module disable postgresql
+			centos_main
+			;;
+		"Red Hat Enterprise Linux")
+			sudo yum -qy module disable postgresql
```

Comment:
Run this command only if, version is >= 8 . This will ensure that the script also runs on versions 7.*.

Reply:
I have introduced the version check in the case of `Red Hat Enterprise Linux`
I am not sure but I don't think version check or disabling the default PG repo will be required in the case of `Red Hat Enterprise Linux Server` as only the rhel7 machine was giving this os name. Let me know if we should include it.

Reply:
Yes, you're right! We don't need the version check for the "Red Hat Enterprise Linux" case. I missed the change where you added another case for RHEL7. Please add a comment making it clear that "Red Hat Enterprise Linux" means RHEL>=8 and ""Red Hat Enterprise Linux Server" means RHEL7.

Reply:
Okay ill do this